README.md

```diff
@@ -142,6 +142,23 @@ steps:
       MERGE_METHOD: MERGE
 ```
 
+### Specify the semantic versioning category to use
```

Comment:
I looked at the specification and didn't see any reference to "category".

I'd suggest refactoring this paragraph to:

```
### Presets

Presets enable additional functionality which can be used to better personalize
default behavior of the Merge me! Action.

Available presets are:
- `DEPENDABOT_MINOR` - Merge only minor dependency updates for pull requests created by Dependabot if the dependency version follows [Semantic Versioning v2](https://semver.org/).
- `DEPENDABOT_PATH` - Merge only patch dependency updates for pull requests created by Dependabot if the dependency version follows [Semantic Versioning v2](https://semver.org/).
```

I think we can drop `DEPENDABOT_MAJOR`, as it literally doesn't change the behavior at all?

Reply:
Great suggestions!

I'll update the README and remove `DEPENDABOT_MAJOR`

Reply:
Done!